src/services/baseService.test.ts

```diff
@@ -21,6 +21,14 @@ class TestService extends BaseService {
     return this.sendFile(requestOptions, filePath);
   }
 
+  public getSlsRegion() {
```

Comment:
Seems like pretty thin wrappers for these two functions. Any reason for having both `getSlsRegion` and `getRegion`?

Reply:
These are purely in a `TestService` class purposely built to test the core/base functionality of the `BaseService`